tools/mOTUs/motus_profiler.xml

```diff
@@ -0,0 +1,254 @@
+                        </when>
+                        <when value="single">
+                            <param name="unpaired_reads" type="data" format="fastq,fastq.gz" label="Unpaired reads" help="-s input file(s) for unpaired reads, fastq(.gz)-formatted"/>
+                        </when>
+                    </conditional>
+                </when>
+                <when value="SAM_BAM_input">
+                    <param name="SAM_BAM_input" type="data" format="sam,bam" label="motus map_tax SAM or BAM file" help="-i SAM or BAM input file(s) generated by motus map_tax"/>
+                </when>
+                <when value="mgc_read_count">
+                    <param name="mgc_count" type="data" format="mgc" label="mgc read count file)(s)" help="-m mgc reads count file generated by motus calc_mgc"/>
+                </when>
+            </conditional>
+                <param name="sample_name" type="text" label="Sample name" help="sample name can contain letters and/or numbers" default_value="unnamed sample"/>
+            <conditional name="db">
+                <param name="db_source" type="select" label="mOTUs Database">
+                    <option value="cached">Use a pre-installed database</option>
+                    <option value="history">Use a database from history</option>
+                </param>
+                <when value="cached">
+                    <param name="db_cached" type="select" label="A pre-installed mOTUs databese" help="no pre-installed mOTUs databases are avalable yet">
+                        <options from_data_table="mOTUs_db">
+                            <column name="value" index="0"/>
+                            <column name="name" index="1"/>
+                            <column name="version" index="2"/>
+                            <column name="path" index="3" />
+                            <filter type="sort_by" column="1"/>
+                        </options>
+                    </param>
+                </when>
+                <when value="history">
+                    <param name="database" type="data" format="tar" label="Database from history" help="Custom database or other version of mOTUs database from link: https://zenodo.org/records/5140350"/>
+                </when>
+            </conditional>
+        </section>
+        <section name="additional" title="Additional" expanded="true">
+            <conditional name="additional_options">
+                <param  name="advanced" type="select" label="Addditional options">
+                    <option value="simple">No, Use param defaults</option>
+                    <option value="advanced">Yes, see full parameter list</option>
+                </param>
+                <when value="advanced">
+                    <param name="ref_motus_only" type="boolean" label="only species with reference genomes" help="-e only species with reference genomes (ref-mOTUs)." truevalue="-e" falsevalue=""/>
```

Comment:
You have truevalue set here ... so you can use this more efficiently in the command section.